docs/application-connector/docs/013-details-api.md

```diff
@@ -27,10 +27,14 @@ kubectl -n kyma-system get svc core-nginx-ingress-controller -o 'jsonpath={.spec
 To access the specification, run:
 
 ```
-curl https://gateway.kyma.local:{NODE_PORT}/{RE_NAME}/v1/metadataapi.yaml
+curl https://gateway.kyma.local:{NODE_PORT}/{RE_NAME}/v1/metadata/api.yaml
 ```
 
 You can acquire the API specification for a given version directly from the Connector Service:
 ```
 curl https://connector-service.{CLUSTER_NAME}.kyma.cx/v1/api.yaml
 ```
+Whereas locally:
```

Comment:
Change to:
`You can also access it locally:`
If that's what you meant 😉

Reply:
Yup, thank you 👍